inference-engine/tests/ie_test_utils/functional_test_utils/src/layer_test_utils/summary.cpp

```diff
@@ -137,7 +147,6 @@ void Summary::saveReport() {
 
     std::string outputFilePath = outputFolder + std::string(CommonTestUtils::FileSeparator) + filename;
 
-    std::vector<ngraph::OpSet> opsets;
     opsets.push_back(ngraph::get_opset1());
```

Comment:
I think it is not correct because if you call `saveReport()` method several times it will extend opsets vector for each call.
It will be better to initialise opsets in the constructor of class.